metrics/api/src/main/java/io/helidon/metrics/api/Meter.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.metrics.api;
+
+/**
+ * Common behavior of all meters.
+ */
+public interface Meter extends Wrapped {
+
+    /**
+     * Unique idenfier for a meter.
+     */
+    interface Id {
+
+        /**
+         * Returns the meter name.
+         *
+         * @return meter name
+         */
+        String name();
+
+        /**
+         * Returns the tags which further identify the meter.
+         *
+         * @return meter tags
+         */
+        Iterable<Tag> tags();
+
+        /**
+         * Unwraps the ID as the specified type.
+         *
+         * @param c {@link Class} to which to cast this ID
+         * @return the ID cast as the requested type
+         * @param <R> type to cast to
+         */
+        default <R> R unwrap(Class<? extends R> c) {
```

Comment:
Should extend `Wrapped` instead?

Reply:
`Meter` already extends `Wrapped`. I forgot to remove this default implementation when I added `Wrapped` and the default implementation there.

Reply:
Fixed.